modules/replicator/src/main/java/org/apache/ignite/internal/replicator/ReplicaService.java

```diff
@@ -55,8 +55,8 @@
 
 /** The service is intended to execute requests on replicas. */
 public class ReplicaService {
-    /** Retry timeout. */
-    private static final int RETRY_TIMEOUT_MILLIS = 10;
+    /** Default timeout for retrying replica operation. */
+    public static final int DEFAULT_REPLICA_OPERATION_RETRY_INTERVAL = 10;
```

Comment:
Move this to ReplicationConfigurationSchema and do not use a constant in the *Schema class.